### ISSUE TITLE
[Feature Request] : Brief description in the character sidebar

### ISSUE THREAD

**Issue by lorffine** [SillyTavern/SillyTavern#547]
Add a text place to show the character's description under the character's name in the sidebar something like:

    Name
    [this is where where the description will show, if there is any]
    [TAGS], [TAGS]

**Comment by kunkanader** [comment]
Yes, this would be very useful for separating forks of the same character. Agnai had this feature for a very long time. 
I'd also like to  propose a simpler approach: make {{char}} placeholder ignore everything after special symbol, like "/" for example. This way user can put short  description of the character inside the name field.
E.g name: "Maria /angry version, likes to break people"
User sees in the character list: "Maria /angry version, likes to break people"
AI sees ( {{char}} placeholder returns ): "Maria"

Yes, the same utility can be achieved by using tags, however, using tags for this purpose can mess up with using tags as actual tags for functions like include/exclude by tag. Tags also can't be used for actual search, since for different character descriptions user may have to use way too many tags for this "search" (that works by "include this tag" function) to be useful at all. And search by name function is already here and can be used with description put inside the name field as I proposed.

**Comment by lorffine** [comment]
Thanks alot for adding the feature! 4230f3881df2b9564bd74ebb8d3d76886500fcb1
But most of the characters description end up missing some informations `...`, and there is no other way to see them, I would suggest wrapping up to a new line, Maybe limit 3~2 lines?

**Comment by lorffine** [comment]
@Cohee1207 Exactly! Thanks for the Amazing work! 👍
